python/ql/lib/semmle/python/security/dataflow/TarSlipCustomizations.qll

```diff
@@ -58,20 +92,36 @@ module TarSlip {
   /**
    * A sink capturing method calls to `extractall`.
    *
-   * For a call to `file.extractall` without arguments, `file` is considered a sink.
+   * For a call to `file.extractall` without `members` argument, `file` is considered a sink.
    */
-  class ExtractAllSink extends Sink {
-    ExtractAllSink() {
-      exists(DataFlow::CallCfgNode call |
-        call =
-          API::moduleImport("tarfile")
-              .getMember("open")
-              .getReturn()
-              .getMember("extractall")
-              .getACall() and
-        not exists(call.getArg(_)) and
-        not exists(call.getArgByName(_)) and
-        this = call.(DataFlow::MethodCallNode).getObject()
+  class ExtractAllwoMembersSink extends Sink {
```

Comment:
This name should probably be written out: `ExtractAllWithoutMembersSink`

Reply:
renamed accordingly